Mlem/App/Models/Action/ActionAppearance+StaticValues.swift

```diff
@@ -204,6 +204,15 @@ extension ActionAppearance {
         )
     }
     
+    static func resolve(isOn: Bool) -> Self {
+        .init(
+            label: isOn ? "Unresolve" : "Resolve",
+            color: isOn ? Palette.main.negative : Palette.main.positive,
+            icon: isOn ? Icons.unresolve : Icons.resolve,
```

Comment:
Imo we shouldn't toggle to `unresolve` when on the bar
```suggestion
            icon: isOn ? Icons.unresolve : Icons.resolve,
            barIcon: isOn ? Icons.resolveFill : Icons.resolve,
```